src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String DEF_FIGURE_COLOR = Color.WHITE.name().toLowerCase();
+    private static final int DEF_RADIUS = 10;
+    private static final int FIGURE_AMOUNT = 5;
+    private static final int MAX_LENGTH = 10;
+
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
```

Comment:
Don't use verbs for class/interface names. While 'FigureSupplier' is not strictly a verb, it might be confused with a method that supplies figures. Consider a name that clearly represents it as a factory or provider of figures, such as 'FigureFactory' or 'FigureProvider'.